.goreleaser.yml

```diff
@@ -22,13 +22,14 @@ archives:
     format_overrides:
       - goos: windows
         format: zip
```

Comment:
@dhingrak do we also need to add the GOEXPERIMENT=boringcrypto environment variable to the goreleaser file? (https://goreleaser.com/customization/env/)

Reply:
sorry, my bad, that link is to do something else. i've just noticed you added it to ci.yml